client/app/scripts/components/node-details.js

```diff
@@ -77,14 +77,19 @@ const NodeDetails = React.createClass({
     const details = this.props.details;
     const nodeExists = this.props.nodes && this.props.nodes.has(this.props.nodeId);
 
+    if (details) {
+      return this.renderDetails();
+    }
+
```

Comment:
undefined is falsey, I think.  nodeExists is true iff the node exists in the currently topology, but the whole point of the bug report was that we don't want the details panel to go empty if the node disappears from the topology (due to a filter).

Do you have other proposals?  Maybe @davkal can suggest a better way?  We don't want node details to appear if the backend reports a 404 for the node.